nx/source/services/nifm.c

```diff
@@ -22,9 +22,14 @@ Result nifmInitialize(void) {
     if (serviceIsActive(&g_nifmSrv))
         return 0;
 
-    Result rc;
-    rc = smGetService(&g_nifmSrv, "nifm:u");
-
+    Result rc = smGetService(&g_nifmSrv, "nifm:s");
```

Comment:
Store the privilege level of the service somewhere in a static var.

**Pinging other developers** to check if nifm:s/nifm:a have a limited number of sessions that would make it a bad idea to init them by default in libnx.

Reply:
What if we used a different init function for :s and :a ? This way it won't break existing homebrew and libnx code

Reply:
That's what sdk-nso has; however I think that's pretty unpleasant and breaks consistency with other service wrappers; for that reason I asked publicly to see if it really breaks anything.